featurebyte/session/snowflake.py

```diff
@@ -311,8 +311,9 @@ def get_columns_schema_from_dataframe(dataframe: pd.DataFrame) -> list[tuple[str
                 db_type = "INT"
             elif (
                 dataframe.shape[0] > 0
-                and dataframe[colname].apply(lambda x: isinstance(x, list)).all()
+                and dataframe[colname].apply(lambda x: x is None or isinstance(x, list)).all()
```

Comment:
nit: Should we use `np.isnan` here?

Reply:
hm seems like a bunch of errors are thrown (https://github.com/featurebyte/featurebyte/actions/runs/6071849693/job/16470734256?pr=1711) if i try to use that. will keep as is for now!